FogCreek/Glitch-Community
# Profuse tea: Move redirects into react, replace withRouter with hooks

## Links
https://profuse-tea.glitch.me/
https://app.clubhouse.io/glitch/story/2952/ssr-should-pick-up-on-react-router-redirects

## Changes:
- Got rid of server/redirects.js, now it's all in pages/router.js
- Redirects from react-router work through server side rendering
- Got rid of all `withRouter` calls, replaced them with the new `useHistory` and `useLocation` hooks

## How To Test:
Try some of the redirected urls and make sure they still work. Of note is the /featured redirect, which should preserve sub paths (so any /culture/* url should work as /featured/*)
